spec/services/export_offenders_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe ExportOffenders do
+  describe '#call' do
+    let(:file) { Rails.root.join('tmp', 'soi_extract.csv') }
+
+    let!(:offender_1) do
```

Comment:
You can probably simplify all the lets from line 7 to line 25 by doing something like the following:

```ruby
let!(:identities) { create_list(:identity, 3) }
```

As long as `noms_offender_id` is defined a sequence in the factory and the `identity` factory creates the offender relationship, you should be good.

You can then just iterate through your `identities` to assert the output.